teradatalabs/centos6-java8-oracle-ldap/files/active-directory-certificate.crt

```diff
@@ -1,23 +1,35 @@
 -----BEGIN CERTIFICATE-----
-MIIDvTCCAqWgAwIBAgIQKa0jt3aw7Y5JEFtf8Vy4kjANBgkqhkiG9w0BAQUFADBl
-MRMwEQYKCZImiZPyLGQBGRYDY29tMRgwFgYKCZImiZPyLGQBGRYIdGVzdGxkYXAx
-FjAUBgoJkiaJk/IsZAEZFgZwcmVzdG8xHDAaBgNVBAMTE3ByZXN0by1BRC1URVNU
-LUNBLTEwHhcNMTYwNjE3MTkyMTA5WhcNMjEwNjE3MTkzMTA5WjBlMRMwEQYKCZIm
-iZPyLGQBGRYDY29tMRgwFgYKCZImiZPyLGQBGRYIdGVzdGxkYXAxFjAUBgoJkiaJ
-k/IsZAEZFgZwcmVzdG8xHDAaBgNVBAMTE3ByZXN0by1BRC1URVNULUNBLTEwggEi
-MA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQCiRZOYgl6+S+2NSOxnkzUKZBew
-AaqNevAajtGIgSXBhcQpC38oAAMWOhCXWEtx+Udzp3qbYpSMMjNTotThhb+EUx+i
-dG5/bpO99qY/6UUAQQL8g2sRyJvQtaTt61ug5hLheaveqBWInfglWV57Pj9++Nc8
-Dcupc9D0+BJ0CT3PE4bjif8ksE5zRdSEkEOcsK4NvCeAgmzJ1AaUXSuI1hl7qNRS
-OEfNoBRPdH5etO/WJvQeHZyOQYRgJwJXhpg7K+je6uyZWIclLMsIZNjs4CrtS9br
-gX6kzeWMPi69fvTR5Yzz8KGmfDpoZk+3IMGo6FrlV2rQYqhr/H1eYXhWHldZAgMB
-AAGjaTBnMBMGCSsGAQQBgjcUAgQGHgQAQwBBMA4GA1UdDwEB/wQEAwIBhjAPBgNV
-HRMBAf8EBTADAQH/MB0GA1UdDgQWBBRhUs5KPw8xotuxOQ34/9HlHAS8wjAQBgkr
-BgEEAYI3FQEEAwIBADANBgkqhkiG9w0BAQUFAAOCAQEACbU08ITkl4NSrIDmReDY
-U0zgnwNlcjyfDQ2eUfnrp3TpUDgGkDGkwAFtjlKngpaEKOWAUUUp/Akr7dKwpmef
-BEuHCqLyZdJB7vQYzcnA9kKgAafinyqSGQVQ3aMYWW/sRdtN2f2STyuGM0x0YeMx
-HuYrqCgoZLUXsyzn8Z1GRRUHSsyd5xdm5vbt5/kPBSV+9mHjS7zCyGJQNdiNWvoY
-jB6aCUs4FN5XgKSenCKg1B92zEJUgI70dKbcZyhzbJxjfnuXbuYNcpCc7ovFSzZg
-HulHmGFygMDnXzK2tzXW2iZ+ypZ8K9qdQeYZxyyiQYotbWPyD5vYMDqJyqv+s8Me
-LA==
+MIIGJTCCBQ2gAwIBAgITHQAAAAUM8gMo/ohRwwAAAAAABTANBgkqhkiG9w0BAQUF
```

Comment:
How long are the new certs good for, and do we have some guidelines for the next time we reissue to reduce the frequency of this problem?